src/components/sidebar/components/sidebar-nav-highlight-item/sidebar-nav-highlight-item.module.css

```diff
@@ -0,0 +1,101 @@
+.root {
+	--inset-border-shadow: inset 0 0 0 1px var(--inset-border-color);
+	--gradient-background: linear-gradient(
+		315deg,
+		var(--gradient-start) 0%,
+		var(--gradient-stop) 100%
+	);
+
+	composes: g-focus-ring-from-box-shadow from global;
+	align-items: center;
+	background: var(--token-color-surface-primary);
+	border-radius: 5px;
+	color: var(--token-color-foreground-primary);
+	display: flex;
+	gap: 8px;
+	margin-bottom: 2px;
+	padding: 8px;
+	position: relative;
+	width: 100%;
+	z-index: 0;
+
+	&:hover {
+		background: var(--gradient-background);
+		color: var(--token-color-foreground-strong);
+	}
+
+	&:focus {
+		z-index: 1;
+	}
+
+	&:active {
+		/* Using box-shadow for an inset border. Also plays nice with focus styles. */
+		box-shadow: var(--inset-border-shadow);
+	}
+
+	/* AKA active */
+	&[aria-current='page'] {
+		background: var(--gradient-background);
+
+		&:not(:focus-visible) {
+			box-shadow: var(--inset-border-shadow);
+		}
+	}
+}
+
+.text {
+	composes: hds-typography-body-200 from global;
+	font-weight: var(--token-typography-font-weight-medium);
+}
+
+/*
+ * Themeing
+ */
+
+.theme-terraform {
+	--gradient-start: var(--token-color-terraform-gradient-faint-stop);
+	--gradient-stop: var(--token-color-terraform-gradient-faint-start);
+	--inset-border-color: var(--token-color-terraform-border);
+}
+
+.theme-packer {
+	--gradient-start: var(--token-color-packer-gradient-faint-stop);
+	--gradient-stop: var(--token-color-packer-gradient-faint-start);
+	--inset-border-color: var(--token-color-packer-border);
+}
+
+.theme-consul {
+	--gradient-start: var(--token-color-consul-gradient-faint-stop);
+	--gradient-stop: var(--token-color-consul-gradient-faint-start);
+	--inset-border-color: var(--token-color-consul-border);
+}
+
+.theme-vault {
+	--gradient-start: var(--token-color-vault-gradient-faint-stop);
+	--gradient-stop: var(--token-color-vault-gradient-faint-start);
+	--inset-border-color: var(--token-color-vault-border);
+}
+
+.theme-boundary {
+	--gradient-start: var(--token-color-boundary-gradient-faint-stop);
+	--gradient-stop: var(--token-color-boundary-gradient-faint-start);
+	--inset-border-color: var(--token-color-boundary-border);
+}
+
+.theme-nomad {
+	--gradient-start: var(--token-color-nomad-gradient-faint-stop);
+	--gradient-stop: var(--token-color-nomad-gradient-faint-start);
+	--inset-border-color: var(--token-color-nomad-border);
+}
+
+.theme-waypoint {
+	--gradient-start: var(--token-color-waypoint-gradient-faint-stop);
+	--gradient-stop: var(--token-color-waypoint-gradient-faint-start);
+	--inset-border-color: var(--token-color-waypoint-border);
+}
+
+.theme-vagrant {
```

Comment:
❓ should we have a generic variant for sidebars like ['well-architected-framework'](https://dev-portal-git-zssidebar-hierarchy-hashicorp.vercel.app/well-architected-framework)? Also, maybe this could be follow up, but we should ask design how to handle that overview item. 

<img width="1107" alt="Screen Shot 2023-01-25 at 1 41 18 PM" src="https://user-images.githubusercontent.com/36613477/214697608-91008bfc-a188-4ce4-9fe5-b09d5f074d12.png">

Reply:
I like this idea! I've stubbed something in for design's consideration. Will add screenshots in a collection "feedback and changes so far" comment 👍

Reply:
Update: have implemented this! ✅